Cargo.toml

```diff
@@ -41,8 +41,8 @@ hyper-util = "0.1"
 
 ic-cdk = "0.13"
 ic-cdk-macros = "0.13"
-ic-agent = "0.35"
-ic-utils = "0.35"
+ic-agent = "0.37.1"
```

Comment:
Is there a reason to lock  this down to the patch version? I generally prefer to keep it more flexible by locking down only to the minor version.

Reply:
I took the same version as used in the dfinity/ic monorepo, but now I specified v0.37.